src/pages/ProductList/Product/Product.scss

```
@@ -0,0 +1,47 @@
+.productParts {
+  display: inline-block;
+  width: 25%;
+  height: 427px;
+  text-align: center;
+  vertical-align: center;
+
+  .listContent {
+    margin: 0 auto;
+
+    .productThumbnail {
+      width: 100%;
+      height: 258px;
+    }
+
+    .productTag {
+      height: 27px;
+
+      .productTagCircle {
+        margin-top: 20px;
+        padding: 3px 17px 3px 17px;
```

아래와 같이 코드를 줄일 수 있습니다! :) 
```suggestion
        padding: 3px 17px 3px;
```